src/traces/scatter/attributes.js

```diff
@@ -65,6 +65,10 @@ module.exports = {
             'See `y0` for more info.'
         ].join(' ')
     },
+    key: {
+        valType: 'data_array',
+        description: 'A list of keys for object constancy of data points during animation'
```

Comment:
Works for me. Yeah, I didn't know quite where to draw the line removing features that will just need to be added right back in on the next step.